### PR TITLE
Update the instructions to run swupd to account for auto-update.

### PR DESCRIPTION
**Pull request by rcaballeromx**
Signed-off-by: Rodrigo Caballero <rodrigo.caballero.abraham@intel.com>

### CODE REVIEW

File: source/clear-linux/how-to/swupd/run.rst [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
--  System is up and running a Clear Linux OS for Intel
-   Architecture official build, 7420 later.
+* The system is up and running the Clear Linux 15400 build or later.
 
--  Device is on a well-connected network.
+* Device is on a well-connected network.
 
--  Device is able to connect to release infrastructure. For example:
-   http://update.clearlinux.org/update/300/
+* Device is able to connect to the release infrastructure. For example:
```
Comment (jodh-intel) [inline source/clear-linux/how-to/swupd/run.rst]: Isn't this almost a duplicate of the previous bullet? Maybe the previous one could be "Device is on a reliable network"? Or maybe just drop the previous point and say, "Device is able to connect to http://update.clearlinux.org/" (the `update/300/` seems superfluous in this context - but if it needs to be retained, maybe change `300` to `15400` as that's mentioned above whilst `300` isn't).

File: source/clear-linux/how-to/swupd/run.rst [hunk 2]
```diff
@@ -1,59 +1,60 @@
...
-This section describes how to manually trigger the software update code in
-Clear Linux\* OS for Intel® Architecture.
+This section describes how to update |CLOSIA|.
 
 Prerequisites
 =============
 
--  System is up and running a Clear Linux OS for Intel
-   Architecture official build, 7420 later.
+* The system is up and running the Clear Linux 15400 build or later.
```
Comment (jodh-intel) [inline source/clear-linux/how-to/swupd/run.rst]: Should this document explain how to establish build (or should that be "release"?) the system is running at currently (`sudo swupd update -s`)?

File: source/clear-linux/how-to/swupd/run.rst [hunk 3]
```diff
@@ -1,59 +1,60 @@
...
+   .. code-block:: console
 
-Application options:
+      $ sudo swupd update
 
-.. code-block:: console
+.. note::
+
+   If the updater console output indicates a kernel update occurred, then at
+   your convenience reboot to begin using the new OS version.
```
Comment (jodh-intel) [inline source/clear-linux/how-to/swupd/run.rst]: I think the wording should be firmer here - kernel updates may well include security fixes so they should be applied as soon as realistically possible and the system rebooted to ensure the system is as secure as possible.

File: source/clear-linux/how-to/swupd/run.rst (hunk 1 above)
Comment (bktan8) [inline source/clear-linux/how-to/swupd/run.rst]: Agree.  Let's change the last 2 lines to simply "Device is able to connect to the release infrastructure http://update.clearlinux.org/.

File: source/clear-linux/how-to/swupd/run.rst (hunk 3 above)
Comment (bktan8) [inline source/clear-linux/how-to/swupd/run.rst]: Agree.  Let's drop "at your convenience".  So it should say something like... "If the swupd console output indicates a kernel update has occurred, then reboot immediately in order for the enhancements to take effect."

File: source/clear-linux/how-to/swupd/run.rst (hunk 2 above)
Comment (rcaballeromx) [inline source/clear-linux/how-to/swupd/run.rst]: I can add the instruction to do so, certainly.

File: source/clear-linux/how-to/swupd/run.rst (hunk 1 above)
Comment (rcaballeromx) [inline source/clear-linux/how-to/swupd/run.rst]: Understood, updating.

File: source/clear-linux/how-to/swupd/run.rst (hunk 3 above)
Comment (rcaballeromx) [inline source/clear-linux/how-to/swupd/run.rst]: I will apply the feedback.

**Comment by jodh-intel** [comment]
lgtm

**Comment by jodh-intel** [comment]
lgtm

**Comment by rcaballeromx** [comment]
Changes will be applied at 6 pm PST

**Pull request merged**